src/display_backend.c

```diff
@@ -54,3 +57,13 @@ bool is_X11_backend() {
     return false;
 }
 
+bool is_Quartz_backend() {
+    GdkDisplay *gdk_display = gdk_display_get_default();
+    #ifdef GDK_WINDOWING_QUARTZ
+    if (GDK_IS_QUARTZ_DISPLAY(gdk_display)) {
+        return true;
+    }
+    #endif
+
+    return false;
+}
```

Comment:
Please fix the no-newline here.

Reply:
You mean delete the newline between `#endif` and `return false;`. Should also change for the functions above?

Reply:
No, there is no newline at the end of this file.